### PR TITLE
Transition to VM weights

### PR DESCRIPTION
**Pull request by oleg-plakida**
- Update weights using VM instances.
 - Change runner tag for weights ci job to weights-vm

cc: paritytech/ci_cd#697

### CODE REVIEW

**Review by sergejparity — APPROVED** [review]

**Comment by oleg-plakida** [comment]
bot bench-vm $ all

**Comment by oleg-plakida** [comment]
bot bench-vm $ all polkadot

**Comment by oleg-plakida** [comment]
bot bench-vm $ all kusama

**Comment by oleg-plakida** [comment]
bot bench-vm $ all kusama

**Comment by oleg-plakida** [comment]
bot bench-vm $ all rococo

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2979903

**Comment by oleg-plakida** [comment]
bot rebase

**Comment by oleg-plakida** [comment]
bot bench-vm $ all rococo

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2981671

**Comment by oleg-plakida** [comment]
bot bench-vm $ all polkadot

**Comment by ggwpez** [comment]
You force pushed to the master. That unfortunately breaks the MR.

**Comment by oleg-plakida** [comment]
bot bench-vm $ all kusama

**Comment by oleg-plakida** [comment]
bot cancel

**Comment by oleg-plakida** [comment]
bot clear

**Comment by oleg-plakida** [comment]
bot bench-vm $ all polkadot

**Comment by oleg-plakida** [comment]
bot bench-vm $ all kusama

**Comment by oleg-plakida** [comment]
bot bench-vm $ all westend

**Comment by oleg-plakida** [comment]
bot bench-vm $ all polkadot

**Comment by oleg-plakida** [comment]
bot bench-vm $ all polkadot

**Comment by oleg-plakida** [comment]
bot bench-vm $ all polkadot

**Comment by oleg-plakida** [comment]
bot bench-vm $ all kusama

**Comment by oleg-plakida** [comment]
bot bench-vm $ all rococo

**Review by chevdor — APPROVED** [review]
That leads to a few major increases, which is fine as long as it is consistent.

https://weights.tasty.limo/compare?old=refs/tags/v0.9.43&new=oleg/ci-cd/weights-vm&threshold=90&unit=weight&repo=polkadot&path_pattern=runtime%2F**%2Fsrc%2Fweights%2F**%2F*.rs&method=guess-worst&ignore_errors=true&git_pull=true

**Review by EgorPopelyaev — APPROVED** [review]

**Comment by ggwpez** [comment]
> That leads to a few major increases, which is fine as long as it is consistent.

Looking at `asymptotic` makes a bit more sense for the ones that do not have a proper base component, but having a few out of ~1670 benchmarks off is fine i think.  
Just need to monitor that they stay consistent after this is merged.

File: runtime/kusama/src/weights/pallet_balances_balances.rs [hunk 1]
```diff
@@ -0,0 +1,154 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```
Comment (ggwpez) [inline runtime/kusama/src/weights/pallet_balances_balances.rs:1]: It seems that we are not using the correct weight file for Kusama follow up for https://github.com/paritytech/polkadot/issues/7408

**Review by ggwpez — APPROVED** [review]

**Review by coderobe — APPROVED** [review]

**Pull request merged**